editor/plugins/shader_editor_plugin.cpp

```diff
@@ -836,50 +837,224 @@ ShaderEditor::ShaderEditor() {
 	_editor_settings_changed();
 }
 
+void ShaderEditorPlugin::_update_shader_list() {
+	shader_list->clear();
+	for (uint32_t i = 0; i < edited_shaders.size(); i++) {
```

Comment:
Doesn't LocalVector have range iterator? Also why is it LocalVector? 🤔

Reply:
I generally don't use the range iterators on arrays. LocalVector is prefered if you are not passing the array around because its simpler and more efficient.